### PR TITLE
fixed navbar absolute 

### PR DESCRIPTION
**Pull request by abhayksahu369**
- fixed navbar absolute.
- created different component for navbar to make code clean.
- added background color and decreased the height of navbar for better ui.

### CODE REVIEW

**Comment by Anu27n** [comment]
Looks fine, please star this repo, I will merge this PR, so it can read to more people.

File: src/style.css [hunk 1]
```diff
@@ -605,8 +605,9 @@ body.nav-active { overflow: hidden; }
   top: 0;
   left: 0;
   width: 100%;
-  background-color: transparent;
-  padding-block: 40px;
+  background-color:var(--smoky-black-2);
```
Comment (Anu27n) [inline src/style.css:608]: reduce the opacity little, its still bit dark, want lighter version.

**Review by Anu27n — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 1 above)
Comment (abhayksahu369) [reply src/style.css:608]: okayy

Comment (Anu27n) [reply src/style.css:608]: It looks fine tho no worries please star this  repo 👍🤞.

Comment (abhayksahu369) [reply src/style.css:608]: thanks brother

Comment (abhayksahu369) [reply src/style.css:608]: @Anu27n please add gssoc-ext and level tag here , score are counted for pr having this tags

**Pull request merged**